cds/modules/previewer/templates/cds_previewer/video/deposit.html

```diff
@@ -33,7 +33,8 @@
   {%- set theo_config = {} -%}
 
   {# Strip eos path for the ``uri`` #}
-  {%- set path = file.file.obj.file.uri | replace(config.VIDEOS_XROOTD_PREFIX, '') %}
+  {%- set path = file.file.obj.file.uri | replace(config.VIDEOS_XROOTD_ENDPOINT, '') |
```

Comment:
If you check one video player in a deposit, currently the `src` path still contains `config.VIDEOS_XROOTD_PREFIX`. It looks to me the `replace` was not working properly even before.
Are you sure that this fix the issue? Theoretically, it does, but how come the `config.VIDEOS_XROOTD_PREFIX` is not replaced with `''`?

Reply:
I found the issue. In the configuration of production we have this:
![screen shot 2018-07-18 at 16 40 10](https://user-images.githubusercontent.com/38131488/42888813-4c3dc3b2-8aa9-11e8-9033-97e82448b196.png)
with test in path instead of prod in config.VIDEOS_XROOTD_PREFIX so the replace could not work. The prefix is created with using config.VIDEOS_LOCATION
Strange thing is that it is taken from config.py not from cds.cfg probably because VIDEOS_XROOTD_PREFIX is composed in config.py so it takes the values defined there, not the ones loaded from cfg.
The conclusion is that this fix will work, in the template config.VIDEOS_XROOTD_ENDPOINT and config.VIDEOS_LOCATION have proper values.

Reply:
I think is better in order to be more clear and follow the invenio way to just compose the config var `VIDEOS_XROOTD_PREFIX` in the `cds.cfg` on each environment and leave it `''` by default. But still if this is happening I think is a problem that should be discussed with invenio guys.